tests/main/interfaces-password-manager-service/task.yaml

```diff
@@ -0,0 +1,51 @@
+summary: Ensure that the password-manager-service interface works
+
+systems:
+    - -ubuntu-core-16-*
+
+prepare: |
+    echo "Ensure we have a working gnome-keyring"
+    apt-get install -y --no-install-recommends gnome-keyring
```

Comment:
I use distro_install_package now, however I am not sure what the packages are called in fedora/suse that I will need there.